packages/core/src/metas/initialize.ts

```diff
@@ -1,44 +1,48 @@
 import type { Config } from '../config';
 import { isFunction } from '../utils';
-import type { Metas, MetasMap } from './types';
+import { VERSION } from '../version';
+import type { Meta, MetaItem, Metas } from './types';
 
 export function initializeMetas(config: Config): Metas {
-  const map: MetasMap = new Map();
+  let items: MetaItem[] = [];
 
-  const add: Metas['add'] = (key, getter) => {
-    if (!map.has(key)) {
-      map.set(key, getter);
-    }
+  const add: Metas['add'] = (item) => {
+    items.push(item);
   };
 
-  const remove: Metas['remove'] = (key) => {
-    map.delete(key);
+  const remove: Metas['remove'] = (item) => {
+    items = items.filter((i) => i !== item);
   };
 
-  add('sdk', () => ({
-    name: '@grafana/javascript-agent-core',
-    version: '0.0.1', // TODO: set correct version here
-  }));
+  const initial: Meta = {
+    sdk: {
+      name: '@grafana/agent-core',
```

Comment:
We can probably pull that from the package json file

Reply:
It's a little bit more difficult. `package.json` is outside of `src` mentioned in `tsconfig.json`. Also it may break once compiled as the path to `package.json` may changed.

I tried it in the early versions of the library.